tests/test_interfaces.py

```diff
@@ -0,0 +1,595 @@
+# pylint: skip-file
+""" This module tests the interface module """
```

Comment:
I'm not sure I understand how you are asserting? I see some methods included from Pyric. Or how you do you mock the interfaces? Whatever you do, please document it.  :)